cityofaustin/atd-moped
# Add Signal IDs and project type columns to project list view

## Associated issues

https://github.com/cityofaustin/atd-data-tech/issues/8930

Adds two new columns to be able to toggle on and off in the project list view. 

Does not add either of them to advance filtering. 

I had to change the sql view to use a common table expression (CTE) because of unexpected repeat type_names based on how many people were in the personnel table. Thank you to @frankhereford for helping me understand what was happening. 

## Testing
**URL to test:** 

test in moped-test or run locally

**Steps to test:**

Add records with signal ids and set project types. The signal IDs column is on by default, the project types column is hidden.  See them in the table and toggle columns on and off. 

![Screen Shot 2022-04-25 at 3 38 47 PM](https://user-images.githubusercontent.com/12474808/165171030-e77d8539-260e-4cf2-99a9-ec94c67428dc.png)


---
#### Ship list
